server/app.js

```diff
@@ -1,5 +1,172 @@
-const express = require('express');
-
+const express = require("express");
+const { MealTimes, Dates } = require("./models/Identifier");
+const { stopWordList, isStopWord, getStopWord } = require("./models/StopWord");
 const app = express();
 
 //TODO: Add an express router
+const router = express.Router();
+
+// Setup middleware so we can read the requests body
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use("/", router); // so router can work
+
+// Set up port and have app listen
+const PORT = ("port", process.env.PORT || 3000);
+app.listen(PORT, () => {
+  console.log(`Listening on port ${PORT}`);
+});
+
+// TASK 1
+router.get("/mealtimes", (req, res) => {
+  if (req.query.query == undefined) {
```

Comment:
`if (req.query.query)` is better.

But Preferred:
`const { query } = req.query;`